operators/vision/draw_bounding_box.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ocos.h"
+#include "string_utils.h"
+
+#include <cstdint>
+
+namespace ort_extensions {
+struct DrawBoundingBox : BaseKernel {
+  DrawBoundingBox(const OrtApi& api, const OrtKernelInfo& info) : BaseKernel(api, info) {
+    thickness_ = TryToGetAttributeWithDefault<int64_t>("thickness", 4);
+    num_classes_ = static_cast<int32_t>(TryToGetAttributeWithDefault<int64_t>("num_classes", 10));
```

Comment:
I expected we'd configure the number of colours to use. If doing output by score, that value is the maximum number of boxes we draw. If the output is by class, that value is the maximum number of classes we can draw results for.

Reply:
Yes, we have 10 of predefined colors to use.
For now, I didn't use `num_classes_`  too much.